.github/ISSUE_TEMPLATE/meeting-tc.md

```diff
@@ -0,0 +1,43 @@
+## Date/Time
+
+| Timezone | Date/Time |
+|----------|-----------|
+<%= [
+  'America/Los_Angeles',
+  'America/Denver',
+  'America/Chicago',
+  'America/New_York',
+  'Europe/London',
+  'Europe/Amsterdam',
+  'Europe/Moscow',
+  'Asia/Kolkata',
+  'Asia/Shanghai',
+  'Asia/Tokyo',
+  'Australia/Sydney'
+].map((zone) => {
+  return `| ${zone} | ${date.setZone(zone).toFormat('EEE dd-MMM-yyyy HH:mm (hh:mm a)')} |`
+}).join('\n') %>
+
+Or in your local time:
+* https://www.timeanddate.com/worldclock/?iso=<%= date.toFormat("yyyy-MM-dd'T'HH:mm:ss") %>
+
+## Agenda
+
+Extracted from **<%= agendaLabel %>** labelled issues and pull requests from **<%= owner %>/<%= repo %>** prior to the meeting.
+
+<%= agendaIssues.map((i) => {
+  return `* ${i.html_url}`
+}).join('\n') %>
```

Comment:
I think that we might want to prevent duplicate items, for reference (https://github.com/expressjs/discussions/issues/208):

![Screenshot 2024-03-04 at 01 12 05](https://github.com/expressjs/discussions/assets/5110813/f51d4ea3-c7ff-4775-859b-abc748b9fecf)

Reply:
I would rather do it in the action than the template. Should be easy enough to add. I'm not sure how the code in place ended with duplicates but it's not a big deal to ignore that unexpected behavior with an explicit check to remove the dupes.

Reply:
I think I fixed that in https://github.com/pkgjs/meet/pull/202